agent/tests/LoopFake.php

```diff
@@ -74,7 +74,7 @@ public function removeWriteStream($stream): void
      */
     public function addTimer($interval, $callback): TimerInterface
     {
-        $frame = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS, 3)[2];
+        $frame = debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS, 2)[1];
```

Comment:
This can remove a frame from the backtrace because we are now using the instance directly, which removes one additional function call from the stack.